packages/mobile/routes/onboarding/views/profile-recovery/ProfileRecoveryRouter.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="typescript">
+    import { setContext } from 'svelte'
+    import { ImportMnemonicPhraseView, SuccessView } from './views'
+    import { localize } from '@core/i18n'
+    import {
+        profileRecoveryRoute,
+        profileRecoveryRouter,
+        ProfileRecoveryRouter,
+        ProfileRecoveryRoute,
+    } from '@core/router'
+    import { showAppNotification } from '@lib/notifications'
+
+    setContext<ProfileRecoveryRouter>('importRouter', $profileRecoveryRouter)
+    $profileRecoveryRouter.resetRoute()
```

Comment:
I personally prefer doing this inside onMount so its more clear when its being triggered